zio-http/src/test/scala/zhttp/http/EncodeClientParamsSpec.scala

```diff
@@ -52,6 +52,20 @@ object EncodeClientParamsSpec extends DefaultRunnableSpec with EncodeClientParam
           val req = encodeClientParams(HttpVersion.HTTP_1_1, params)
           assert(req.headers().getInt(HttpHeaderNames.CONTENT_LENGTH).toLong)(equalTo(5L))
         }
+      } +
+      testM("host header") {
+        check(anyClientParam) { params =>
+          val req        = encodeClientParams(HttpVersion.HTTP_1_1, params)
+          val reqHeaders = req.headers()
+          val hostHeader = HttpHeaderNames.HOST
+
+          params.url.host match {
```

Comment:
Remove this logic from test,

This can be split into two test
1.  Client Params with absolute URL and Host Header both Passed.
2. Client Params with absolute URL without Host Header.